tests/unit/plugins/lookup/test_bitwarden.py

```diff
@@ -0,0 +1,167 @@
+                "linkedId": None,
+                "name": "a_new_secret",
+                "type": 1,
+                "value": "this is a new secret"
+            },
+            {
+                "linkedId": None,
+                "name": "not so secret",
+                "type": 0,
+                "value": "not secret"
+            }
+        ],
+        "folderId": "3b12a9da-7c49-40b8-ad33-aede017a7ead",
+        "id": "90992f63-ddb6-4e76-8bfc-aede016ca5eb",
+        "login": {
+            "password": "passwordA3",
+            "passwordRevisionDate": "2022-07-26T23:03:23.399Z",
+            "totp": None,
+            "username": "userA"
+        },
+        "name": "a_test",
+        "notes": None,
+        "object": "item",
+        "organizationId": None,
+        "passwordHistory": [
+            {
+                "lastUsedDate": "2022-07-26T23:03:23.405Z",
+                "password": "a_new_secret: this is secret"
+            },
+            {
+                "lastUsedDate": "2022-07-26T23:03:23.399Z",
+                "password": "passwordA2"
+            },
+            {
+                "lastUsedDate": "2022-07-26T22:59:52.885Z",
+                "password": "passwordA"
+            }
+        ],
+        "reprompt": 0,
+        "revisionDate": "2022-07-26T23:03:23.743Z",
+        "type": 1
+    },
+    {
+        "collectionIds": [],
+        "deletedDate": None,
+        "favorite": False,
+        "folderId": None,
+        "id": "5ebd4d31-104c-49fc-a09c-aedf003d28ad",
+        "login": {
+            "password": "b",
+            "passwordRevisionDate": None,
+            "totp": None,
+            "username": "a"
+        },
+        "name": "dupe_name",
+        "notes": None,
+        "object": "item",
+        "organizationId": None,
+        "reprompt": 0,
+        "revisionDate": "2022-07-27T03:42:40.353Z",
+        "type": 1
+    },
+    {
+        "collectionIds": [],
+        "deletedDate": None,
+        "favorite": False,
+        "folderId": None,
+        "id": "90657653-6695-496d-9431-aedf003d3015",
+        "login": {
+            "password": "d",
+            "passwordRevisionDate": None,
+            "totp": None,
+            "username": "c"
+        },
+        "name": "dupe_name",
+        "notes": None,
+        "object": "item",
+        "organizationId": None,
+        "reprompt": 0,
+        "revisionDate": "2022-07-27T03:42:46.673Z",
+        "type": 1
+    }
+]
+
+
+class MockBitwarden(Bitwarden):
+
+    logged_in = True
+
+    def _get_matches(self, search_value, search_field="name"):
+        return list(filter(lambda record: record[search_field] == search_value, MOCK_RECORDS))
+
+
+class LoggedOutMockBitwarden(MockBitwarden):
+
+    logged_in = False
+
+
+class TestLookupModule(unittest.TestCase):
+
+    @patch('ansible_collections.community.general.plugins.lookup.bitwarden._bitwarden', new=MockBitwarden())
+    def test_bitwarden_plugin_no_match(self):
+        lookup_plugin = LookupModule()
```

Comment:
Check out one of the tests in `tests/unit/plugins/lookup/` that uses `lookup_loader`. That will take care of properly loading the plugin so that the config manager will work.